src/ride/vehicle.c

```diff
@@ -150,13 +150,54 @@ const uint8 _soundParams[SOUND_MAXID][2] = {
 	{ 0, 0 }	// SOUND_62
 };
 
+bool vehicle_move_info_valid(int cd, int typeAndDirection, int offset)
```

Comment:
If cd is an index into gTrackVehicleInfo, maybe it can be called trackVehicleInfoId or something?